Library/Formula/vala.rb

```diff
@@ -3,8 +3,8 @@
 class Vala < Formula
   desc "Compiler for the GObject type system"
   homepage "https://live.gnome.org/Vala"
-  url "http://ftp.gnome.org/pub/GNOME/sources/vala/0.26/vala-0.26.1.tar.xz"
-  sha256 "8407abb19ab3a58bbfc0d288abb47666ef81f76d0540258c03965e7545f59e6b"
+  url "http://ftp.gnome.org/pub/GNOME/sources/vala/0.29/vala-0.29.1.tar.xz"
```

Comment:
This is an odd version number, meaning it’s a [development release](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27909/version=mavericks/testReport/junit/brew-test-bot/mavericks/audit_vala/).